### ISSUE TITLE
Create a Gitter channel

### ISSUE THREAD

**Issue by lolmaus** [alexanderGugel/ied#20]
Could be https://gitter.im/alexanderGugel/ied

**Comment by alexanderGugel** [comment]
Great idea! Not sure about Gitter vs Slack though.

**Comment by alexanderGugel** [comment]
Added in 798d14f71c113c3481efbe1545663a73f14b7c16

**Comment by lolmaus** [comment]
Gitter is the way to go. It reuses Github login, while Slack requires a new login/password combination for every community. Also, Slack has ridiculous pricing and free plan drops history.

**Comment by alexanderGugel** [comment]
Ok, I'm a bit lost to be honest. I just created the [channel](https://gitter.im/alexanderGugel/ied?utm_source=share-link&utm_medium=link&utm_campaign=share-link), but it honestly looks a bit empty.

Is "just adding" users really the proper way to get people to use it? Seems a bit spammy to me tbh.

I just signed up for example and am already in a ton of channels....

**Comment by lolmaus** [comment]
Nah, don't try inviting people. A readme badge is enough. People will come by when they want to ask/discus something.

You don't have to keep the channel open all the time. You'll receive an email shortly after someone posts to the channel.
